### PR TITLE
Initial Prettier fixes.

### PR DESCRIPTION
**Pull request by KatieProchilo**
The CLI, service, and the top-level project have two new scripts, and a change to the existing `build` script:
* `npm run build` will also run `npm run check`.
* `npm run check` lists any files that are not up to Prettier's standards.
* `npm run fix` will make changes to bring the code up to Prettier's standards.

Probably a good time to introduce this change as it touches a lot of lines and no one's been pushing to d-ser-t. All Prettier config has been set within `package.json` for each package.

Also manually alphabetized imports.

### CODE REVIEW

File: packages/d-ser-t-service/package.json [hunk 1]
```diff
@@ -1,52 +1,60 @@
...
+    },
+    "devDependencies": {
+        "@types/diff": "^4.0.2",
+        "@types/jest": "^24.0.15",
+        "jest": "^24.8.0",
+        "prettier": "1.18.2",
+        "ts-jest": "^24.0.2"
+    },
+    "scripts": {
+        "build": "tsc --build && npm run check",
```
Comment (Joll59) [inline packages/d-ser-t-service/package.json]: As we are enforcing these style rules, the build step should fix before building.

**Comment by Joll59** [comment]
The build step is currently failing because running the build steps before the prettier fixes results in Prettier errors. 
Also if we run a full lint comnnd `tslint -c tslint.json 'src/**/*.ts'` command there are some linting errors that are out standing.

File: packages/d-ser-t-service/package.json (hunk 1 above)
Comment (KatieProchilo) [reply packages/d-ser-t-service/package.json]: Resolved with https://github.com/Joll59/d-ser-t/commit/4ce1909bffe64446bb8e9d9281085e8a4c383140.

Comment (KatieProchilo) [reply packages/d-ser-t-service/package.json]: Resolved with https://github.com/Joll59/d-ser-t/commit/4ce1909bffe64446bb8e9d9281085e8a4c383140.

**Comment by KatieProchilo** [comment]
Ready to be looked at

**Pull request merged**